<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdWidgetQt4</class>
 <widget class="QWidget" name="ThresholdWidgetQt4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Segment tissues based on thresholding techniques.</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="mModeGroupBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="mManualModeRadioButton">
          <property name="text">
           <string>Manual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mHistoModeRadioButton">
          <property name="text">
           <string>Histo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mKMeansRadioButton">
          <property name="text">
           <string>k-means</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mEMModeRadioButton">
          <property name="text">
           <string>EM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QStackedWidget" name="mModeStackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="mManualWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="mManualNrTissuesLabel">
               <property name="text">
                <string>#Tissues</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="mManualNrTissuesSpinBox">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>30</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mManualLimitNrLabel">
               <property name="text">
                <string>Limit-Nr:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="mManualLimitNrSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="mLoadBordersPushButton">
               <property name="text">
                <string>Open...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mSaveBordersPushButton">
               <property name="text">
                <string>Save...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="mManualThresholdLabel">
               <property name="text">
                <string>Threshold:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mThresholdLowerLabel">
               <property name="text">
                <string>xxxxxxx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="mThresholdHorizontalSlider">
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mThresholdUpperLabel">
               <property name="text">
                <string>xxxxxxx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mThresholdBorderLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mHistoWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="mSubsectionGroupBox">
             <property name="title">
              <string>Subsection</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>px:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="mHistoPxSpinBox"/>
                </item>
               </layout>
              </item>
              <item row="0" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>py:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="mHistoPySpinBox"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>lx</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="mHistoLxSpinBox"/>
                </item>
               </layout>
              </item>
              <item row="1" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>ly:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="mHistoLySpinBox"/>
                </item>
               </layout>
              </item>
              <item row="0" column="3">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="mMinPixelsFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="mHistoMinPixelsLabel">
                <property name="text">
                 <string>Min. Pixels:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mHistoMinPixelsSpinBox">
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="mHistoMinPixelsRatioLabel">
                <property name="text">
                 <string>Ratio:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="mHistoMinPixelsRatioHorizontalSlider">
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>160</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mKmeansEMWidget">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="mKMeansNrTissuesLabel">
                 <property name="text">
                  <string>#Tissues</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="mKMeansNrTissuesSpinBox">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="value">
                  <number>2</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="mKMeansDimsLabel">
                 <property name="text">
                  <string>#Dims</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="mKMeansDimsSpinBox">
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="mKMeansImageNrLabel">
                 <property name="text">
                  <string>Image-Nr:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="mKMeansImageNrSpinBox">
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>13</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="mKMeansWeightLabel">
               <property name="text">
                <string>Weight:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="mKMeansWeightHorizontalSlider">
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="mKMeansFilenameFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="mKMeansFilenameLabel_2">
                <property name="text">
                 <string>Filename:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mKMeansFilenameLineEdit_2"/>
              </item>
              <item>
               <widget class="QPushButton" name="mKMeansFilenamePushButton_2">
                <property name="text">
                 <string>Select</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="mChannelFrame_2">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="mKMeansRRadioButton_2">
                   <property name="text">
                    <string>R</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="mKMeansGRadioButton_2">
                   <property name="text">
                    <string>G</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="mKMeansBRadioButton_2">
                   <property name="text">
                    <string>B</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="mKMeansARadioButton_2">
                   <property name="text">
                    <string>A</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="mKMeansIterationsFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="mKMeansIterationsLabel">
                <property name="text">
                 <string>#Iterations:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mKMeansIterationsSpinBox">
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="mKMeansConvergeLabel">
                <property name="text">
                 <string>Converge:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mKMeansConvergeSpinBox">
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="singleStep">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QCheckBox" name="mUseCenterFileCheckBox">
           <property name="text">
            <string>Use InitCenters:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mCenterFilenameLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mSelectCenterFilenamePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QCheckBox" name="mAllSlicesCheckBox">
           <property name="text">
            <string>Apply to all slices</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mExecutePushButton">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
